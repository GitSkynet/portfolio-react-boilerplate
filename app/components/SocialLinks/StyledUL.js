import styled from 'styled-components';

const StyledUL = styled.ul`
  align-items: center;
  display: flex;
  list-style: none;
  margin: 0;
  padding: 0;

  & li:not(:first-child) {
    margin-left: 1rem;
  }
`;

export default StyledUL;
