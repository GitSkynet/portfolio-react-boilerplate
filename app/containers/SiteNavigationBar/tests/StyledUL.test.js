import React from 'react';
import { shallow } from 'enzyme';

import StyledUL from '../StyledUL';

describe('<StyledUL />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = shallow(<StyledUL />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
