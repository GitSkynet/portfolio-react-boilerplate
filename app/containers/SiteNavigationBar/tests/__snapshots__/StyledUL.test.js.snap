// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledUL /> should render and match the snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <ul
    className="StyledUL__StyledUL-kEPlCt cXXRcw"
/>,
  "nodes": Array [
    <ul
      className="StyledUL__StyledUL-kEPlCt cXXRcw"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <StyledUL__StyledUL />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "cXXRcw",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <ul
          className="StyledUL__StyledUL-kEPlCt cXXRcw"
/>,
        "_debugID": 2,
        "_renderedOutput": <ul
          className="StyledUL__StyledUL-kEPlCt cXXRcw"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <StyledUL__StyledUL />,
}
`;
